templates/web/base/hercules/bin_days.html

```diff
@@ -0,0 +1,76 @@
+[% SET bodyclass = 'hercules' %]
+[% USE date(format = '%A, %-d~~~ %B') %]
+
+[% INCLUDE header.html %]
+
+<h1 class="govuk-heading-xl">Your bin days</h1>
+
+<dl class="hercules__address">
+  <dt class="hercules__address__title">Address</dt>
+  <dd class="hercules__address__property">[% property.Description | title %]</dd>
+</dl>
+<div class="hercules__collections">
+  <h2 class="govuk-heading-l">Your collections</h2>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">  
+
+[% FOR unit IN service_data %]
+
+<a href="#" class="hercules-service-name-link"><h3 class="govuk-heading-m">[% unit.service_name %]</h3></a>
+
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Frequency</dt>
+    <dd class="govuk-summary-list__value">[% unit.schedule %]</dd>
+  </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Next collection</dt>
+    <dd class="govuk-summary-list__value">
+      [% IF unit.next %]
+        [% date.format(unit.next) | replace('~~~', unit.next_ordinal) %]
+        [% IF unit.next_changed %]Adjusted[% END %]
+      [% ELSE %]
+        <i>None</i>
+      [% END %]
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Last collection</dt>
+    <dd class="govuk-summary-list__value">
+      [% IF unit.last %]
+        [% date.format(unit.last) | replace('~~~', unit.last_ordinal) %]
+      [% ELSE %]
+        <i>None</i>
+      [% END %]
+    </dd>
+  </div>
+</dl>
+
+<div class="hercules-services-launch-panel">
+    [% INCLUDE hercules/services.html %]
```

Comment:
If you miss that you can click on the heading to expand the list of options there's no way to carry out some of the services in here.